### DISCUSSION TITLE
Support for Micronaut plugin #9429 without profiling

### DISCUSSION THREAD

**Discussion by skundu1234** [apache/skywalking#10122] [General]
Thanks for micronaut plugin.We can see now traces,endpoints,metrics but profiling is not working for that but it should work in 9.2.0 and skywalking java 8.12 agent.
Can you please suggest that profiling is still out of scope for micronuat application or we can expect that with next release.

**Comment by wu-sheng** [comment]
I think there is nothing that changed for Agent profiling. As a new plugin is introduced, you should check what is the new endpoint for profiling.

**Comment by skundu1234** [comment]
We have checked below PR but did not see anything about profiling.we have tested our micronaut applications we have hit multiple times Post API and traces are also coming during profiling time and its more than 10ms in traces but still unable to see. But we can see profiling for springboot application.

https://github.com/apache/skywalking/pull/9455

**Reply by wu-sheng** [reply]
>  traces are also coming during profiling time and its more than 10ms

Profiling typically is not suitable for a 10-20ms request. It is too fast. The minimal profiling period is 10ms, it doesn't mean it can profile a request with 10ms latency. Typically, profiling is for requests in over 500ms, which is a real slow request.
If you use profiling to measure 10ms request, the profiling itself costs more than the method itself.

**Reply by skundu1234** [reply]
Thanks but we have tested for some of the trace is more than 2000 ms but still unable to see in profiling.Can you please check mentioned PR and guide [pg-yang](https://github.com/pg-yang) to test profiling for micronaut application.

**Reply by wu-sheng** [reply]
I don't think the plugin has anything related to profiling. pg-yang is a committer of the community. He is one of the owner, there is nothing for me to guide him. This is not how open source works.

**Reply by skundu1234** [reply]
Sure Thanks @wu-sheng .
@pg-yang  can you please test profiling for your micronaut application

**Reply by pg-yang** [reply]
Profiling is provided by ` core` rather than plugins, so the PR doesn't have code about it.
Did you check the logs of agent? Please provide the details about this issue.

**Reply by Arunkumarlnr** [reply]
@pg-yang  @skundu1234 We don't have any error in the agent logs.. Also our sample request is over 500ms but still we couldn't see profiling details for this application.
[skywalking-api.log](https://github.com/apache/skywalking/files/10182791/skywalking-api.log)
<img width="341" alt="Trace details" src="https://user-images.githubusercontent.com/41669194/206377603-62b51ac8-f2bc-41ce-8904-17b46d33c956.png">
<img width="669" alt="Profile Details" src="https://user-images.githubusercontent.com/41669194/206378182-af8cec8a-8c65-4d77-9a6a-f91b9b6f15c7.png">

**Reply by pg-yang** [reply]
<img width="1257" alt="image" src="https://user-images.githubusercontent.com/3917424/206404048-f634087e-9847-4396-845d-149ab1e48c99.png">

I don't think the plugin has anything related to profiling either. Anyway,  I couldn't reproduce the issue.
